iaas/ec2/task.go

```diff
@@ -55,14 +55,19 @@ func (t *ec2WaitTask) Run(job monsterqueue.Job) {
 	var notifiedSuccess bool
 	t0 := time.Now()
 	for {
+		if time.Now().Sub(t0) > time.Duration(2*timeout)*time.Second {
```

Comment:
if `timeout` is 0 there is a chance that task will not run. Is this a expected behavior?